polkadot/runtime/parachains/src/configuration/migration/v10.rs

```diff
@@ -0,0 +1,325 @@
+					assert_eq!(v9.validation_upgrade_cooldown              , v10.validation_upgrade_cooldown);
+					assert_eq!(v9.validation_upgrade_delay                 , v10.validation_upgrade_delay);
+					assert_eq!(v9.max_pov_size                             , v10.max_pov_size);
+					assert_eq!(v9.max_downward_message_size                , v10.max_downward_message_size);
+					assert_eq!(v9.hrmp_max_parachain_outbound_channels     , v10.hrmp_max_parachain_outbound_channels);
+					assert_eq!(v9.hrmp_sender_deposit                      , v10.hrmp_sender_deposit);
+					assert_eq!(v9.hrmp_recipient_deposit                   , v10.hrmp_recipient_deposit);
+					assert_eq!(v9.hrmp_channel_max_capacity                , v10.hrmp_channel_max_capacity);
+					assert_eq!(v9.hrmp_channel_max_total_size              , v10.hrmp_channel_max_total_size);
+					assert_eq!(v9.hrmp_max_parachain_inbound_channels      , v10.hrmp_max_parachain_inbound_channels);
+					assert_eq!(v9.hrmp_channel_max_message_size            , v10.hrmp_channel_max_message_size);
+					assert_eq!(v9.code_retention_period                    , v10.code_retention_period);
+					assert_eq!(v9.on_demand_cores                          , v10.on_demand_cores);
+					assert_eq!(v9.on_demand_retries                        , v10.on_demand_retries);
+					assert_eq!(v9.group_rotation_frequency                 , v10.group_rotation_frequency);
+					assert_eq!(v9.paras_availability_period                , v10.paras_availability_period);
+					assert_eq!(v9.scheduling_lookahead                     , v10.scheduling_lookahead);
+					assert_eq!(v9.max_validators_per_core                  , v10.max_validators_per_core);
+					assert_eq!(v9.max_validators                           , v10.max_validators);
+					assert_eq!(v9.dispute_period                           , v10.dispute_period);
+					assert_eq!(v9.no_show_slots                            , v10.no_show_slots);
+					assert_eq!(v9.n_delay_tranches                         , v10.n_delay_tranches);
+					assert_eq!(v9.zeroth_delay_tranche_width               , v10.zeroth_delay_tranche_width);
+					assert_eq!(v9.needed_approvals                         , v10.needed_approvals);
+					assert_eq!(v9.relay_vrf_modulo_samples                 , v10.relay_vrf_modulo_samples);
+					assert_eq!(v9.pvf_voting_ttl                           , v10.pvf_voting_ttl);
+					assert_eq!(v9.minimum_validation_upgrade_delay         , v10.minimum_validation_upgrade_delay);
+					assert_eq!(v9.async_backing_params.allowed_ancestry_len, v10.async_backing_params.allowed_ancestry_len);
+					assert_eq!(v9.async_backing_params.max_candidate_depth , v10.async_backing_params.max_candidate_depth);
+					assert_eq!(v9.executor_params						   , v10.executor_params);
+				    assert_eq!(v9.minimum_backing_votes					   , v10.minimum_backing_votes);
+				    assert_eq!(v9.client_features					       , v10.client_features);
```

Comment:
i am confused, how does it compile if v9 shouldn't have this field?

Reply:
actually, I copy-pasted this test from other config migrations and it is testing almost nothing as it is :)
I reworked it and I'll post it soon